### ISSUE TITLE
Blaze Spawner issues

### ISSUE THREAD

**Issue by darkeshrine** [SnowShock35/Obscurity#84]
I'm playing on the Recommended, 1.1.11, version of the pack on the FTB launcher. I'm setting up my first mob spawner and the blaze spawner recieved inside of the compact machine for the hell quest doesn't seem to spawn blazes, at least not normal blazes. Even when it's placed on Extra Utilities Cursed Earth, it doesn't seem to be spawning blazes. I have had A blaze spawn, but it was about 60 blocks lower in elevation from the spawner and is immortal. Granted, the blaze doesn't seem to "see" me so It's basically docile, but I can't hurt it either.

**Comment by SnowShock35** [comment]
Will look into this ;)

**Comment by tibblaye** [comment]
yes i also have this problem i have not had a blaze spawn from it

**Comment by SnowShock35** [comment]
@tibblaye What is the difficulty on?

**Comment by darkeshrine** [comment]
Mine is on whatever difficulty the default for the pack is. The cursed earth spawns mobs just fine. I haven't been able to spawn in another spawner to check if it's that one, the world, etc. NEI has mob spawners locked and I can't seem to make them show up.

**Comment by tibblaye** [comment]
im on normal

**Comment by SnowShock35** [comment]
@tibblaye @darkeshrine Is the spawner in the cube?

**Comment by darkeshrine** [comment]
Mine was not in a compact machine. It was in a cobble box with cursed earth underneath it in the deep dark.

**Comment by SnowShock35** [comment]
Strange :/

**Comment by johnnymobbins** [comment]
Can we get the one in the cube working? or the abiliy to get it working once out of the cube?

**Comment by LyraelRayne** [comment]
Created a new world and had a poke at the spawner. Its nbt data doesn't look right.
![image](https://cloud.githubusercontent.com/assets/11632412/8751184/ed0d202e-2cee-11e5-80e3-64a608029ebe.png)

Moved it out into the deep dark with a diamond dolly, same deal.

![image](https://cloud.githubusercontent.com/assets/11632412/8751223/19919dc8-2cef-11e5-808b-47ae4ad9f28c.png)


Edited the values to match a skeleton spawner I found in one of the fortresses and blazes started popping out.

![image](https://cloud.githubusercontent.com/assets/11632412/8751339/afdfaba8-2cef-11e5-965e-cb1fa8bb8bc2.png)

**Comment by SnowShock35** [comment]
Hmm okay. This will be fixed in 1.2.0 > but for now there isn't anything we can do, sorry :/

**Comment by LyraelRayne** [comment]
Cool. For anyone else reading this, this is the mod I used to fix my spawner http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1286750-in-game-nbtedit-edit-mob-spawners-attributes-in . Install the mod, load the game, point the cursor at your spawner and run the /nbtedit command.  Edit the values to be the same as the ones in my screenshot (except x,y,z) and it'll fix your spawner. You edit a value by clicking on the value and clicking the second icon from the right on the top row. Don't forget to press save once it's done.

**Comment by johnnymobbins** [comment]
Thanks Lyrael, thats great info.

**Comment by masshuku** [comment]
For my self as a quick fix, I just cheated in a spawner, using this command to spawn one at my feet:

/setblock ~ ~ ~ mob_spawner 0 destroy {EntityId:Blaze}
